docs/examples/swift-api-walkthrough/api-walkthrough/ViewController.swift

```diff
@@ -281,12 +283,42 @@ class ViewController: UIViewController {
             
         }
     }
-
-    override func didReceiveMemoryWarning() {
-        super.didReceiveMemoryWarning()
-        // Dispose of any resources that can be recreated.
+    
+    func inOperatorQuery() {
+        let countries = ["France", "United States"]
+        let query = Query.select(
+                SelectResult.expression(Expression.property("name"))
+            )
+            .from(DataSource.database(database!))
+            .where(
+                Expression.property("country").in(countries)
+                .and(Expression.property("type").equalTo("airline"))
+            )
+        do {
+            for row in try query.run() {
+                print("\(row.toDictionary())")
+            }
+        } catch let error {
+            print(error.localizedDescription)
+        }
     }
 
-
+    func inOperatorSatisfiesQuery() {
+        let query = Query.select(
+                SelectResult.expression(Expression.property("sourceairport"))
+            )
+            .from(DataSource.database(database!))
+            .where(
+                Expression.any("FLIGHT_VAR").in(Expression.property("schedule"))
+                    .satisfies(Expression.variable("FLIGHT_VAR.flight").equalTo("AF103"))
+        )
```

Comment:
@pasin this query doesn't succeed currently. Error message in console is:

```
Invalid variable name 'FLIGHT_VAR.flight'
```

Document schema:

```
["sourceairport": ROR, "id": 9920, "airline": AF, "distance": 3213.192432728316, "stops": 0, "destinationairport": NRT, "schedule": <__NSArrayM 0x618000050500>(
{
    day = 0;
    flight = AF832;
    utc = "21:20:00";
},
{
    day = 0;
    flight = AF043;
    utc = "16:08:00";
})
```

Reply:
https://github.com/couchbase/couchbase-lite-ios/issues/1875

Reply:
@jamiltz The issue was fixed in the feature/2.0 branch.